vmdb/app/models/ems_refresh/parsers/kubernetes.rb

```diff
@@ -44,7 +44,18 @@ def process_collection_item(item, key)
 
     def parse_node(node)
       new_result = parse_base_item(node)
-      # TODO, add more properties such as resources
+
+      node_memory = node.status.capacity.send(:memory) || nil
+      node_memory = human_to_value(node_memory) unless node_memory.nil?
+
+      new_result.merge!(
+        :capacity_num_cores => node.status.capacity.send(:cpu) || nil,
+        :capacity_memory    => node_memory,
```

Comment:
All that data is in the hardware table.  You would need a reference to `has_one :hardware` like we have in vm_or_template.rb